xapian-bindings/java/SmokeTest.java

```diff
@@ -142,6 +154,15 @@ public static void main(String[] args) throws Exception {
 		System.err.println("Unexpected docid");
 		    System.exit(1);
 	    }
+	    // New iterator, should start from the beginning 
```

Comment:
Yikes, I apologize for all of the spacing issues. I will avoid them from now onward. I ran the test before committing but I guess I must've added that inverted test case afterward without checking. 
I'll fix it up and check it properly.